SessionMessagingKit/Open Groups/OpenGroupManagerV2.swift

```diff
@@ -107,6 +120,20 @@ public final class OpenGroupManagerV2 : NSObject {
         Storage.shared.removeV2OpenGroup(for: thread.uniqueId!, using: transaction)
     }
     
+    // MARK: Block
+    private var blocklist: [String] = {
+        let path = Bundle.main.path(forResource: "blocklist", ofType: "txt")!
+        let content = try! String(contentsOfFile: path)
```

Comment:
Do we want the force-unwrap and force-try here? (not sure if we want it to crash if the file is removed or becomes invalid - especially if we end up automating the updating of the file)

Reply:
True, I'll change it. We don't want any crash.